### ISSUE TITLE
ProteomicsLFQ: add output of IDs after protein inference and filtering

### ISSUE THREAD

**Issue by hendrikweisser** [OpenMS/OpenMS#6093]
The ProteomicsLFQ tool performs protein inference and FDR filtering of peptide/protein IDs (before quantification). It would be helpful to have an optional output (idXML file) of the IDs after these steps, e.g. for QC purposes or further processing on the ID level.

**Comment by hendrikweisser** [comment]
See also https://github.com/nf-core/quantms/issues/27.

**Comment by jpfeuffer** [comment]
Thanks. We will move development to the nf-core repo soon, so this is a good place.

**Comment by hendrikweisser** [comment]
> does this correspond to one of the files written if debug > 666 ?

Not quite, but possibly the `inferred_protein_ids`/`inferred_peptide_ids` at the end of (or directly after) `inferProteinGroups_`. But I'm not intimately familiar with the control flow in ProteomicsLFQ, so your assessment would be helpful.

**Comment by jpfeuffer** [comment]
I think Hendrik is right. It is the merged IDs plus inference (i.e. with groups and scores)

**Comment by jpfeuffer** [comment]
Recent changes made this suggestion invalid.
We perform inference and FDR based on the consensusXML now (including unassigned peptides, so basically ignoring quantification).
All the information should be in the final consensusXML now.

One thing we could still think of, is to perform FDR calculation even if no filtering is intended. Then, if someone needs the full data they just run with FDR=1 and filter later on by themselves.

**Comment by timosachsenberg** [comment]
stale and probably resolved
